service/src/main/kotlin/fi/espoo/evaka/attendance/ChildAttendanceController.kt

```diff
@@ -341,7 +341,7 @@ private fun Database.Read.getAttendancesResponse(unitId: UUID): AttendanceRespon
         )
     }
 
-    return AttendanceResponse(unitInfo, children)
+    return AttendanceResponse(unitInfo, children.sortedBy { it.firstName })
```

Comment:
I think others have encouraged sorting in frontend. You can use lodash `_,sortyBy(children, child => child.name)`

Reply:
Moved the sorting to the frontend.